### PR TITLE
3.21.0

### PR DESCRIPTION
**Pull request by phansys**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.21.0](https://github.com/sonata-project/SonataBlockBundle/compare/3.20.0...3.21.0) - 2020-08-25
+### Added
+- [[#732](https://github.com/sonata-project/SonataBlockBundle/pull/732)] Added support for "twig/twig:^3.0" ([@phansys](https://github.com/phansys))
+- [[#730](https://github.com/sonata-project/SonataBlockBundle/pull/730)] Added support for symfony/options-resolver:^5.1 ([@phansys](https://github.com/phansys))
+
+### Fixed
+- [[#717](https://github.com/sonata-project/SonataBlockBundle/pull/717)] Missing `?` in type hints where null is default value ([@wbloszyk](https://github.com/wbloszyk))
```
Comment (VincentLanglet) [inline CHANGELOG.md:11]: This is pedantic, no ?

Comment (wbloszyk) [reply CHANGELOG.md:11]: IMHO it is not pedantic becouse with `?` people can call methods with full available param even if param is null what was unavailable before #717 PR

**Review by jordisala1991 — APPROVED** [review]

**Review by wbloszyk — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (VincentLanglet) [reply CHANGELOG.md:11]: I'm not sure to understand. The `?` is not needed when `null` is the default value.
```
function foo(string $f = null) {}

foo();
foo(null);
```
Both work.

Comment (wbloszyk) [reply CHANGELOG.md:11]: Agree, I think about other PR where default value is not null. In this PR all change have `= null` so it is pedantic.

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**